package.json

```diff
@@ -198,7 +198,7 @@
     "eslint-import-resolver-webpack": "0.8.3",
     "eslint-plugin-import": "2.8.0",
     "eslint-plugin-jsx-a11y": "6.0.2",
-    "eslint-plugin-lodash": "2.4.3",
+    "eslint-plugin-lodash": "2.5.0",
```

Comment:
Was this dependency update required for anything? I'm happy for eslint plugins to be updated as part of this PR, just interested to know why it was done ;)

Reply:
yarn complained about it being a different version from the one in app/package.json...